build/js/validation.js

```diff
@@ -0,0 +1,149 @@
+'use strict';
+
+function validateNames(firstName, lastName) {
+    // checks if name fields are not empty and dont contain digits
+    var containsNumbers = function containsNumbers(nameToTest) {
+        return (/[0-9]/.test(nameToTest)
+        );
+    };
+
+    if (containsNumbers(firstName) || containsNumbers(lastName)) {
```

Comment:
RegExp evaluation is more costly than length comparison, so you should test length before digits, to get better perfs.